.changeset/tame-grapes-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/math-input": minor
+---
+
+Stop bundling mathquill CSS - enables consuming app to control bundling and font resolution
```

Comment:
I'm guessing we need to let Learning Equality know?

Reply:
Actually I'm a little confused by this changeset. Here you're saying "enables consuming app to control bundling and font resolution" but in the comment below you say "This resolve is for the fonts in Mathquill. We want to bundle them into the math-input package so that consumers of @khanacademy/math-input don't need to _also_ depend on the mathquill package." The two comments seem at odds, right?

Reply:
It gives consumers control by bundling the Mathquill fonts into the `@khanacademy/math-input` package complete with correct CSS files point to the bundled font files using relative paths. 

The custom resolve was for Mathquill fonts, but after you asked, I found I was able to simplify things and didn't need a custom resolver (the built-in one handles things fine). And that was exactly the motivation for updating Mathquill! Thanks.